packages/compiler-cli/ngcc/src/dependencies/dependency_resolver.ts

```
@@ -81,7 +81,7 @@ export class DependencyResolver {
 
     let sortedEntryPointNodes: string[];
     if (target) {
-      if (target.compiledByAngular) {
+      if (target.compiledByAngular && graph.hasNode(target.path)) {
```

If we have specifically requested a target entry-point then this **should** be a fatal error, not silently ignored.
Currently we crash, instead we should log a useful error explaining which dependency is missing.